apps/user-office-backend/src/queries/AdminQueries.ts

```diff
@@ -94,17 +95,60 @@ export default class AdminQueries {
         ];
 
       const proto = Object.getPrototypeOf(element);
-      const names = Object.getOwnPropertyNames(proto).filter(
-        (item) => item !== 'constructor'
-      );
+      if (!proto.constructor.name.startsWith('Admin')) {
```

Comment:
Would be nice to have a comment on why is are mutations starting with Admin excluded